src/video/windows/SDL_windowsevents.c

```diff
@@ -40,6 +40,11 @@
 /* For GET_X_LPARAM, GET_Y_LPARAM. */
 #include <windowsx.h>
 
+/* For WM_TABLET_QUERYSYSTEMGESTURESTATUS et. al. */
+#if _WIN32_WINNT >= _WIN32_WINNT_VISTA
+#include <tpcshrd.h>
+#endif // _WIN32_WINNT >= _WIN32_WINNT_VISTA
```

Comment:
You can add #ifndef XXX / #define XXX for the definitions you need instead of bumping the WINNT version needed.

Reply:
I think a better solution would be a check for `tpcshrd.h` in autotools
(configure.ac) and in cmake script (CMakeLists.txt): It would define the
macro `HAVE_TPCSHRD_H` if the header is found, and you can change your
cpp checks to use that. For standalone headers, e.g. `SDL_config_windows.h`
(and _maybe_ `SDL_config_winrt.h`?) can define it unconditionally: VS2010
and later come with Vista+ SDKs by default, yes?

Reply:
Oy...all that, just to avoid bumping _WINNT_WIN32 to a more recent _unsupported_ version of MS Windows?

@sezero : Thank you for that patch...I wouldn't have figured out the autoconf variant any time soon.

My latest push to the branch uses HAVE_TPCSHRD_H .
I figure the patch can be committed on top of my branch, so that the proper credit is given.